src/migration.ts

```diff
@@ -58,6 +58,14 @@ const normalizeRequestOptions = ({ updatedOnly, payloadMode, ...actual }: Outdat
 export const getMigratedOptions = (panel: PanelModel<OutdatedPanelOptions>): PanelOptions => {
   const { ...options } = panel.options;
 
+  if (!!options.elements?.length) {
+    options.elements.forEach((element) => {
+      if (element.type === FormElementType.DATETIME && !element.hasOwnProperty('isUseLocalTime')) {
+        element.isUseLocalTime = false;
```

Comment:
do we need migration? by default it will be an undefined which is falsy. Otherwise, please replace mutation to a solution with immutable update

Reply:
updated